src/test/java/ladder/domain/LineTest.java

```diff
@@ -0,0 +1,38 @@
+package ladder.domain;
+
+import static ladder.domain.StepStatus.EXIST;
+import static ladder.domain.StepStatus.NONE;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class LineTest {
+
+    @DisplayName("인원수 -1 개수 만큼의 StepStatus들을 가진 Line 인스턴스를 생성한다.")
+    @Test
+    void createLine() {
+        final Line line = new Line(() -> true, 4);
+
+        assertThat(line.getLineResult().value().size())
+                .isEqualTo(4 - 1);
```

Comment:
```suggestion
        assertThat(line.getLineResult().value().size())
                .isEqualTo(3);
```
assertion에서 검증하는 대상에 대해서 지양하는편이에요. 검증 대상에 연산이 들어갔을 때 그 연산이 틀릴 수도 있고 무슨 값을 기대하고 있느냐를 한번 더 알아봐야하기 때문에 연산이 없는 특정한 검증대상을 넣는 것이 좋습니다.

Reply:
바로 반영하였습니다!
감사합니다 ☺️